### ISSUE TITLE
feat: dynamic coverage summary path

### ISSUE THREAD

**Issue by Narixius** [jpb06/jest-badges-action#8]
Hi dear maintainer
Thanks for this great solution.

I am using this action on a monorepo project, so jest generates multiple coverage-summery files per package. I just wanted to suggest that is there a way to make coverageSummaryPath dynamic? then users can run this action multiple times for every coverage-summery file that is generated by jest.

I checked the path of the coverage-summary.json file and I found out that it is statically sat on the project.
https://github.com/jpb06/jest-badges-action/blob/43b8143fa6eff15cfabe7f765fbd3e200b64201a/src/logic/jest/isJestCoverageReportAvailable.ts#L11

Is there a way to make it dynamic, and let the user define this path on the action yml file?
like the code below
```
  - name: Generating coverage badges
    uses: jpb06/jest-badges-action@v1.6.0
    summery-path: packages/main/coverage/coverage-summary.json
    with:
      branches: master
``` 

Thanks for maintaining this project.

**Comment by jpb06** [comment]
Hello!

Please accept my apologies. I missed this issue as I was heading to holidays grounds!

I created a PR to answer your need: https://github.com/jpb06/jest-badges-action/pull/9

The idea is, as you suggested, to add an optional input to the action: `coverage-summary-path`, that would default to `./coverage/coverage-summary.json`.

Would that suit your needs?

**Comment by jpb06** [comment]
Released as [1.8.0](https://github.com/marketplace/actions/jest-badges-generation-action).
Let me know how it rolls!
